src/providers/hooks.ts

```diff
@@ -1,6 +1,16 @@
-import { useContext } from 'react';
+import { useContext, useMemo } from 'react';
 import { OrganizationContext, UserContext } from './contexts';
 
 export const useUser = () => useContext(UserContext);
 
+/**
+ * Default time zones until we have a provider.
+ */
+export const useTimeZones = () =>
+  useMemo(() => {
+    const defaultTimeZones = (Intl as any).supportedValuesOf('timeZone').map((tz: any) => ({ id: tz, longName: tz }));
```

Comment:
moving this into the function seems to fix the tests error.. not sure how related.
will get this in for now and see how it works.